pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Print the lines where a specific column is numerically equal to a given number:
+
+`tsv-filter -H --eq {{field_name}}:{{number}} {{path/to/tsv_file}}`
+
+- Print the lines where a specific column is [eq]ual/[n]on [e]qual/[l]ess [t]han/[l]ess than or [e]qual/[g]reater [t]han/[g]reater than or [e]qual to a given number:
+
+`tsv-filter --{{eq|ne|lt|le|gt|ge}} {{column_number}}:{{number}} {{path/to/tsv_file}}`
+
+- Display the lines where a specific column is [eq]ual/[n]ot [e]qual/part of/not part of a string:
+
+`tsv-filter --str-{{eq|ne|in-fld|not-in-fld}} {{column_number}}:{{string}} {{path/to/tsv_file}}`
+
+- Filter for non-empty fields:
+
+`tsv-filter --not-empty {{column_number}} {{path/to/tsv_file}}`
+
+- Display fields that are empty in a specific column:
```

Comment:
I don't understand the output of this example.

Reply:
You are right, this was wrongly worded by me

```suggestion
- Print the lines where a specific column is empty:
```